app/stac_api/management/commands/perftest.py

```diff
@@ -0,0 +1,118 @@
+import datetime
+import logging
+import random
+
+from dateutil.parser import isoparse
+
+from django.contrib.gis.geos import Polygon
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.core.management.base import BaseCommand
+
+from stac_api.models import Asset
+from stac_api.models import Collection
+from stac_api.models import Item
+from stac_api.validators import MEDIA_TYPES
+
+# path definition relative to the directory that contains manage.py
+logger = logging.getLogger(__name__)
+
+XMIN = 2570000
+# this would be the max extent for CH
+# XMAX = 2746000
+XMAX = 2580000
+YMIN = 1130000
+# this would be the max extent for CH
+# YMAX = 1252000
+YMAX = 1140000
```

Comment:
Maybe a better short comment on the algorithm that generates the geometry for the items here would be good, how many item would be generated ? Why these numbers ? If I understand your comment, some item would be out of switzerland ? Sorry I'm a totally beginner with geo stuff :-P